index.js

```diff
@@ -125,9 +130,12 @@ program
 
         const prevContributions = await tryAndWait(sequencer, sessionID);
         const receipt = await runCeremony(sequencer, sessionID, entropy, prevContributions);
+        const receiptJson = JSON.stringify(receipt, null, '\t');
 
-        // TODO: Store this receipt?
-        console.log(receipt);
+        fs.writeFilesy('receipt.json', receiptJson, (err) => {
```

Comment:
Maybe this should be fixed like below.
```suggestion
        fs.writeFileSync('receipt.json', receiptJson, (err) => {
```

Reply:
[3cb4236](https://github.com/dsrvlabs/czg-keremony/pull/29/commits/3cb42361e44f4948b690a17511f17caf42f2cc67)

Resolved! :)